Sources/Nitrozen-SwiftUI/Elements/Stepper/NitrozenStepperView.swift

```diff
@@ -0,0 +1,194 @@
+//
+//  NitrozenStepperView.swift
+//  
+//
+//  Created by Rushang Prajapati on 24/04/23.
+//
+
+import Foundation
+import SwiftUI
+
+public struct NitrozenStepperView: View {
+    
+    @Binding var inputValue:String
+    var appearance: NitrozenAppearance.StepperView
+    var range: Range<Int>
+    var step:Int
+    var itemSpacing: CGFloat
+    var maxInputValue:Int
+    var minInputValue:Int
+    var intInputValue:Int {
+        return Int($inputValue.wrappedValue) ?? 0
+    }
+    
+    public init(
+        inputValue: Binding<String>,
+        range:Range<Int>,
+        step:Int,
+        itemSpacing:CGFloat,
+        appearance: NitrozenAppearance.StepperView? = nil
+
+    ) {
+        self._inputValue = inputValue
+        self.appearance = appearance.or(NitrozenAppearance.shared.stepperView)
+        self.range = range
+        self.step = step
+        self.itemSpacing = itemSpacing
+        self.maxInputValue = self.range.upperBound - 1
+        self.minInputValue = self.range.lowerBound
+    }
+    
+    public var body: some View {
+        HStack(spacing: self.itemSpacing){
+            decrementAction()
```

Comment:
rename to decrementActionButton()
decrementAction() indicates execution of some action